lib/pengine/pe_output.c

```diff
@@ -2418,13 +2418,18 @@ resource_list(pcmk__output_t *out, va_list args)
         } else if (pcmk_is_set(show_opts, pcmk_show_brief) && (rsc->variant == pe_native)) {
             continue;
 
-        /* Skip resources that aren't at least partially active,
-         * unless we're displaying inactive resources
+        /* Skip resources that are managed but aren't at least partially active,
+         * unless we're displaying inactive resources.
          */
-        } else if (!partially_active && !pcmk_is_set(show_opts, pcmk_show_inactive_rscs)) {
+        } else if (pcmk_is_set(rsc->flags, pe_rsc_managed) && !partially_active &&
+                   !pcmk_is_set(show_opts, pcmk_show_inactive_rscs)) {
```

Comment:
I think the ->active() method is the right place to make a change rather than this function. That might have dramatic repercussions but it would be worthwhile knowing what those are, it could be an improvement. If we need to limit the impact we can promote the gboolean all to an int and add a value for what we want.

Reply:
I am actually surprised at how little the test results changed.  So, either it's alright or we need better tests.

Reply:
Wow, much cleaner. I think it's good